desktop/core/src/desktop/urls.py

```diff
@@ -274,3 +274,7 @@
   urlpatterns += [
     url(r'^oidc/', include('mozilla_django_oidc.urls')),
   ]
+
+urlpatterns += [
```

Comment:
Let's add a config flag to globally turn on/off the Slack integration.

We could do a new [[slack]] section in [desktop], similarly to https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/conf.py#L685

Reply:
Added the config flag. Please let me know if my variable naming or messages (like help etc.) are good enough.

Reply:
Added the config flag